src/main/java/net/wurstclient/altmanager/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package net.wurstclient.altmanager;
+
+public class AuthenticationException extends Throwable {

```

Comment:
Doesn't AuthenticationException already exist in default java packages?

Reply:
It does but it's related to SSL

Reply:
Ok